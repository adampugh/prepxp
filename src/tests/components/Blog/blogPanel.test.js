import React from "react";
import { shallow } from "enzyme";
import BlogPanel from "../../../components/Blog/blogPanel";
import posts from "../../fixtures/blogPosts";

describe("<blogPost />", () => {
    test("should match snapshot", () => {
        const wrapper = shallow(<BlogPanel post={posts[0]} />);
        expect(wrapper).toMatchSnapshot();
    });
});