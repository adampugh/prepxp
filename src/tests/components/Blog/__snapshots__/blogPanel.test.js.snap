// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<blogPost /> should match snapshot 1`] = `
<div
  className="blog__post"
>
  <img
    alt="blog post"
    src="http://via.placeholder.com/300x300"
  />
  <div
    className="blog__post__text"
  >
    <h2>
      5 Tips to get things done
    </h2>
    <p />
  </div>
</div>
`;
